src/main/java/org/grnet/pidmr/service/DatabaseProviderService.java

```diff
@@ -420,9 +421,14 @@ private void checkIfActionsSupported(Set<String> actions) {
     @Transactional
     public AdminProviderDto updateProviderStatus(Long id, ProviderStatus status) {
 
-        var provider = providerRepository.findById(id);
-        provider.setStatus(status);
+        var newProvider = providerRepository.findById(id);
+        var userID = newProvider.getCreatedBy();
+        EmailContextForStatusUpdate emailContext = new EmailContextForStatusUpdate(userID, keycloakAdminService.getUserEmail(userID), newProvider.getId(), String.valueOf(status));
 
-        return ProviderMapper.INSTANCE.databaseAdminProviderToDto(provider);
+        mailerService.sendEmailsWithContext(emailContext, MailType.PROVIDER_ADMIN_ALERT_CHANGE_PID_TYPE_ENTRY_REQUEST_STATUS,MailType.PROVIDER_ADMIN_ALERT_CHANGE_PID_TYPE_ENTRY_REQUEST_STATUS);
```

Comment:
when a provider type request status is updated 2 emails are sent because in the  mailerService.sendEmailsWithContext the mails go first to the admins and then to the users. we should have a case to distinguish the update(where only the user should receive emails) from the create of the pid type (where both admin and user should receive emails).